parts/outgoing_nozzle_common.go

```diff
@@ -274,10 +244,16 @@ func newBatch(cap_count uint32, cap_size uint32, logger *log.CommonLogger) *data
 		capacity_count:    cap_count,
 		capacity_size:     cap_size,
 		getMeta_map:       make(base.McRequestMap),
-		bigDoc_noRep_map:  make(map[string]NeedSendStatus),
+		noRep_map:         nil,
```

Comment:
Why nil? Wouldn't be safer to do make(map) here to avoid panic when accessing nonavailable map?
Given the Clone() method will always clone one too

Reply:
In xmem_nozzle.go, lines 1093 and 1130, noRep_map will be assigned a map. So there is no need to do a make here because it will be overwritten.